docs/website/src/styles/base/reset.scss

```diff
@@ -0,0 +1,20 @@
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default margin */
+body,
```

Comment:
I read an article recently about using `:where(h1, h2, h3) { margin: 0; }` etc. https://css-tricks.com/using-the-specificity-of-where-as-a-css-reset/